data/en/test_preview.json

```diff
@@ -0,0 +1,166 @@
+{
+    "data_version": "0.0.1",
+    "description": "UK Innovation Survey",
+    "groups": [{
+            "blocks": [{
+              "id": "introduction",
+              "type": "Introduction",
+              "body": [{
+                "type": "intro-preview",
```

Comment:
We have a convention of using upper camel case for types (to distinguish them from ids), so we could have `Introduction`, `Basic` and `Definition`